roles/common/tasks/main.yml

```diff
@@ -38,6 +38,26 @@
   vars:
     hosts_filename: "/etc/hosts"
 
+- name: Set default umask to 0022 in /etc/login.defs
+  lineinfile:
+    dest: /etc/login.defs
+    regexp: '^UMASK'
+    line: 'UMASK           022'
+    state: present
+
+- name: Configure creation of user home directories to be user group readable only
+  lineinfile:
+    dest: "/etc/adduser.conf"
+    regexp: "^DIR_MODE="
+    line: "DIR_MODE=0750"
+    state: present
+
+- name: Ensure the system user's home directory is private
+  file:
+    path: "/home/{{ ansible_user }}"
```

Comment:
what is the purpose of this task? ubuntu won't always be the ansible user. When playbooks are run from jenkins tasks the ansible user is jenkins_bot.

Reply:
This is just to ensure the system user (ubuntu) home directory is private as per AARNet's security concerns. I didn't realise the issue with jenkins_bot. I can change it to /home/ubuntu

Reply:
Does aarnet want the mode of /home/ubuntu to be 0700?

Reply:
That is my memory of the discussion with Ollie. Their whole rationale for making everyone's home directories private was because of access to the system user (ubuntu) home directory being open by default. Compromise is that we make it private, mitigating some of their security concerns.

Reply:
I can't see anything in /home/ubuntu on aarnet that would need to be r'd or x'd by anyone other than ubuntu.